src/avram/box.cr

```diff
@@ -37,4 +39,22 @@ abstract class Avram::Box
   def self.create_pair
     2.times { new.create }
   end
+
+  # Returns a value with a number to use for unique database values.
```

Comment:
I think you can remove “database” since the values are unique whether used for database values or not

Reply:
Oh huh, I thought it was database constraint-based. 👍